code/controllers/subsystems/ticker.dm

```diff
@@ -274,15 +274,15 @@ Helpers
 
 /datum/controller/subsystem/ticker/proc/create_characters()
 	for(var/mob/new_player/player in global.player_list)
-		if(player && player.ready && player.mind)
-			if(player.mind.assigned_role=="AI")
-				player.close_spawn_windows()
-				player.AIize()
-			else if(!player.mind.assigned_role)
+		if(player.ready && player.mind)
+			if(!player.mind.assigned_role)
+				continue
+			var/mob/living/newplayer = player.create_character()
+			if(newplayer.mind.assigned_job.do_spawn_special(newplayer, player, FALSE))
+				qdel(player)
 				continue
 			else
-				if(player.create_character())
-					qdel(player)
+				qdel(player)
```

Comment:
I may be misreading the diff here, but as the clause following this is an `else if`, this seems equivalent to
```
newplayer.mind.assigned_job.do_spawn_special(newplayer, player, FALSE)
qdel(player)
```
(from line 281-285).